### PR TITLE
Add rafael/langtonant to dev/langonant

### PR DESCRIPTION
**Pull request by Rafasebass25**
This a little advance. I am still working in the excercise

### CODE REVIEW

**Review by moreHc — DISMISSED** [review]

File: katas/langtonant/rafael/module.js [hunk 1]
```diff
@@ -0,0 +1,70 @@
+
+import { ShowMatrix } from "./module1.js";
```
Comment (J0c4) [inline katas/langtonant/rafael/module.js:2]: function names are good, module names should be improved

File: katas/langtonant/rafael/module.js [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+ {
+    console.log('Matrix showed is ' + Area + ' * '+Area);
+    console.log('Number of steps: '+Steps);
+
+
+    let extension = parseInt(Area);
+    let Picture=new Array();
+
+
+    for(let i=0;i<extension;i++)
```
Comment (J0c4) [inline katas/langtonant/rafael/module.js:23]: There are lint issues, please fix them

**Review by J0c4 — CHANGES_REQUESTED** [review]
You have nice function names, and logic seems to be working fine. Lint issues need to be fixed and modules names

**Comment by J0c4** [comment]
Also, please don't forget to fix merge conflicts

**Pull request merged**